README.md

```diff
@@ -411,6 +412,23 @@ class { 'collectd::plugins::memcached':
 }
 ```
 
+
+See [collectd-haproxy](https://github.com/signalfx/integrations/blob/master/collectd-haproxy/10-haproxy.conf) for configurable parameters.
+
+####Class: `collectd::plugin::haproxy`
+
+```puppet
+class { 'collectd::plugins::haproxy':
+  modules => {
+    'config' => {
+      'Socket'  => '\"/var/lib/haproxy/stats\"',
```

Comment:
Don't make people have to do this nested quoting, just add in the quotes in the erb template.